helpers/screenSize.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+/**
+ * Get viewport screen size.
+ * @example
+ *  {{screenSize}}
+ */
+const factory = () => {
+    return function() {
+        return window.outerWidth;
```

Comment:
window object is available only on the browser, while handlebars is being run on the server. How this can help reduce code bundled code?